docs/README.md

```diff
@@ -1,3 +1,42 @@
 # VTEX Responsive Layout
 
-Under construction.
+Responsive-layout allows you to declare layout structures that will only be rendered in a specific screen-size breakpoint.
+
+There are four blocks defined and exported by this app:
+
+- `responsive-layout.desktop`
+- `responsive-layout.mobile`
+- `responsive-layout.tablet`
+- `responsive-layout.phone`
+
+Each of them have `composition: children`, which means that they expect to receive an array of `children` blocks to be rendered by them, if the current screen-size is right for their breakpoint.
+
+## Blocks API
+
+These blocks are defined by the interfaces below:
+
+```json
```

Comment:
I think this part of our docs is useless.

I would jump right into some examples of usage.